lib/DriverDisplay/DriverDisplayC.cpp

```diff
@@ -52,26 +55,25 @@ CONSTANT_MODE constant_drive_mode = CONSTANT_MODE::SPEED;
 bool constant_drive_set = false;
 //=======================================
 
-DriverDisplayC *DriverDisplayC::_instance = 0;
+// DriverDisplayC *DriverDisplayC::_instance = 0;
 DriverDisplayC::DISPLAY_STATUS DriverDisplayC::status = DISPLAY_STATUS::SETUP;
 
 string DriverDisplayC ::getName() { return "Display0 (driver display)"; };
 
 void DriverDisplayC ::init() {
   abstract_task::init();
+  status = DISPLAY_STATUS::DISPLAY_CONSOLE;
   re_init();
-  status = DISPLAY_STATUS::DISPLAY_DEMOSCREEN;
 }
 
-void DriverDisplayC ::re_init(void) {
-  _setup();
-  status = DISPLAY_STATUS::DISPLAY_BACKGROUND;
-}
+void DriverDisplayC ::re_init(void) { _setup(); }
 
 void DriverDisplayC ::_setup() {
   xSemaphoreTake(spiBus.mutex, portMAX_DELAY);
   tft = Adafruit_ILI9341(SPI_CS_TFT, SPI_DC, SPI_MOSI, SPI_CLK, SPI_RST, SPI_MISO);
   tft.begin();
+  // read for bugs: https://forums.adafruit.com/viewtopic.php?p=815969
+  // tft.initSPI(3000000, SPI_MODE3);
```

Comment:
Is this workaround now needed or not? If not, we might want to move this to a Q&A section in the docs